probes/src/main/java/cloud/fogbow/probes/core/probes/FogbowServiceSuccessRateProbe.java

```diff
@@ -16,19 +16,22 @@
 import org.springframework.stereotype.Component;
 
 /**
- * FogbowServiceSuccessRateProbe is responsible for measuring the success rate in requesting a resource. The success rate is
- * measured by the relationship between the number of orders that failed after requests ({@link OrderState#FAILED_AFTER_SUCCESSFUL_REQUEST}) and the
- * number of orders opened ({@link OrderState#OPEN}).
+ * FogbowServiceSuccessRateProbe is responsible for measuring the success rate in requesting a
+ * resource. The success rate is measured by the relationship between the number of orders that
+ * failed after requests ({@link OrderState#FAILED_AFTER_SUCCESSFUL_REQUEST}) and the number of
+ * orders opened ({@link OrderState#OPEN}).
  */
 @Component
 public class FogbowServiceSuccessRateProbe extends Probe {
 
-    private static final String PROBE_LABEL = "service_success_rate";
+    private static final String PROBE_NAME = "service_success";
```

Comment:
Keep the `service_success_rate`. It's more descriptive.

Reply:
Done.